### ISSUE TITLE
Support a psuedo-reset button

### ISSUE THREAD

**Issue by unwiredben** [robinhedwards/UnoCart-2600#6]
A feature that seems doable would be to have a pseudo-reset button on the board hooked up to a GPIO and pull-up resistor which would cause the firmware to start sending a series of BRK instructions until the button was released, upon which the cart would start serving the menu program.  This would save wear-and-tear on the power switch on the console and allow switching games quicker.

**Comment by DFL-git** [comment]
Agree, a reset button will be great. The STM32 has an NRST line and maybe can be used to restart the main program. I'm not an expert on A2600, but if I understood, the unwiredben's post is to take care of the initial code vector, right?

**Comment by unwiredben** [comment]
Yes, it was a way to getting the currently running 6502 code on the 2600 back into a known state, although that might not be sufficient to act like a just powered-on 2600.  Most games will init everything to the state they want, but I know some make assumptions about the state of RAM that wouldn't hold in this situation.
